client/src/js/components/bhEntitySelect/bhEntitySelect.tmpl.html

```diff
@@ -0,0 +1,32 @@
+<div ng-form="EntityForm" bh-entity-select ng-model-options="{ updateOn: 'default' }">
+  <div
+    class="form-group"
+    ng-class="{ 'has-error' : $ctrl.validateTrigger && EntityForm.entity_uuid.$invalid }">
+
+    <label ng-hide="$ctrl.label" class="control-label" translate>
+      ENTITY.LABEL
+    </label>
+
+    <label ng-show="$ctrl.label" class="control-label" translate>
+      $ctrl.label
```

Comment:
Shouldn't this be `{{ $ctrl.label }}`?

Reply:
Instead of having the `ngShow`/`ngHide` logic, it seems more natural to have `{{ $ctrl.label }}` and set `$ctrl.label` to be a default value of 'ENTITY.LABEL' in the controller.  That way, we only have one `<label>` element in the HTML.